frontend/src/components/DeviceList/DeviceList.tsx

```diff
@@ -6,70 +6,101 @@ import { DeviceSetupItem } from '../DeviceSetupItem'
     registeredId: state.backend.device.uid,
     loggedInUser: getAccountId(state) === state.auth.user?.id,
     myDevice: getOwnDevices(state).find(device => device.id === state.backend.device.uid),
   }))
 
+  const css = useStyles()
+
   return (
-    <>
-      <List>
-        {registeredId ? (
-          loggedInUser &&
-          (myDevice ? (
-            <>
-              <DeviceListItem
-                key={registeredId}
-                device={myDevice}
-                connections={connections[registeredId]}
-                thisDevice={true}
-                setContextMenu={setContextMenu}
-              />
-              <Divider />
-            </>
+    <div className={css.root}>
+      <div className={css.deviceList}>
+        <List>
+          {registeredId ? (
+            loggedInUser &&
+            (myDevice ? (
+              <>
+                <DeviceListItem
+                  key={registeredId}
+                  device={myDevice}
+                  connections={connections[registeredId]}
+                  thisDevice={true}
+                  setContextMenu={setContextMenu}
+                />
+                <Divider />
+              </>
+            ) : (
+              <>
+                <ListItem>
+                  <Notice>This device is registered to another user</Notice>
+                </ListItem>
+                <Divider />
+              </>
+            ))
           ) : (
             <>
               <ListItem>
                 <Notice>This device is not registered to you.</Notice>
               </ListItem>
+              <DeviceSetupItem />
               <Divider />
             </>
-          ))
-        ) : (
-          <>
-            <DeviceSetupItem />
-            <Divider />
-          </>
-        )}
-        {devices.map(
-          device =>
-            device.id !== myDevice?.id && (
-              <DeviceListItem
-                key={device.id}
-                device={device}
-                connections={connections[device.id]}
-                setContextMenu={setContextMenu}
-              />
+          )}
+          {devices.length ? (
+            devices.map(
+              device =>
+                device.id !== myDevice?.id && (
+                  <DeviceListItem
+                    key={device.id}
+                    device={device}
+                    connections={connections[device.id]}
+                    setContextMenu={setContextMenu}
+                  />
+                )
             )
-        )}
-      </List>
-      <LoadMore />
-      <ServiceContextualMenu
-        el={contextMenu.el}
-        serviceID={contextMenu.serviceID}
-        setEl={el => setContextMenu({ ...contextMenu, el })}
-      />
-    </>
+          ) : (
+            <Body center>
+              <Typography variant="body1" color="textSecondary">
+                Not devices found
+              </Typography>
+            </Body>
```

Comment:
```suggestion
```

Check these edits, I think they are probably pulled from an older version of the file and aren't right.